tests/ha/iscsi_client.pm

```diff
@@ -17,14 +17,17 @@ use Utils::Backends qw(is_remote_backend);
 use utils qw(zypper_call systemctl);
 use testapi;
 use hacluster;
-use version_utils 'is_sle';
+use version_utils qw(is_sle);
 
 sub run {
     # Some remote backends connect to the root-console via sshXtermVt or ipmiXtermVt,
     # which set DISPLAY and cause yast2 to show its graphical version. This unsets
     # DISPLAY so the terminal version is shown instead when testing in textmode
     assert_script_run 'unset DISPLAY' if (check_var('DESKTOP', 'textmode') && is_remote_backend());
 
+    # Restart udevd in 15-SP3+ if hostname was changed. See comment 15 in bsc#1177927 for details
+    systemctl 'restart systemd-udevd' if (is_sle('15-SP3+') && get_var('HOSTNAME'));
```

Comment:
I added the fix only for 15-SP3+ as we are currently only seeing the drbd issue in that version, but I also think that a restart of udevd at this point in the test is not really harmful, so we could do it always and for all versions. WDYT @ldevulder @juadk ?

Reply:
Fine for me, and it may appears in post 15-SP3 as well in a maintenance package.

Reply:
Yes, we can do it for all versions.

Reply:
> Yes, we can do it for all versions.

Done.